docs/usage.md

```diff
@@ -24,28 +24,41 @@ Testcontainers will try to connect to a Docker daemon using the following strate
 ### Usage modes
 
 * [Temporary database containers](usage/database_containers.md) - specialized Microsoft SQL Server, MariaDB, MySQL, PostgreSQL, Oracle XE and Virtuoso container support
+* [Elasticsearch container](usage/elasticsearch_container.md) - Elasticsearch container support
 * [Webdriver containers](usage/webdriver_containers.md) - run a dockerized Chrome or Firefox browser ready for Selenium/Webdriver operations - complete with automatic video recording
 * [Kafka containers](usage/kafka_containers.md) - run a dockerized Kafka, a distributed streaming platform
 * [Generic containers](usage/generic_containers.md) - run any Docker container as a test dependency
 * [Docker compose](usage/docker_compose.md) - reuse services defined in a Docker Compose YAML file
 * [Dockerfile containers](usage/dockerfile.md) - run a container that is built on-the-fly from a Dockerfile
 
-## Maven dependencies
+## Gradle/Maven dependencies
 
-Testcontainers is distributed in a handful of Maven modules:
+Testcontainers is distributed in a handful of Gradle/Maven modules:
 
 * **testcontainers** for just core functionality, generic containers and docker-compose support
 * **mysql**, **postgresql** or **oracle-xe** for database container support
+* **elasticsearch** for elasticsearch container support
 * **selenium** for selenium/webdriver support
 * **nginx** for nginx container support
 
 In the dependency description below, replace `--artifact name--` as appropriate and `--latest version--` with the [latest version available on Maven Central](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.testcontainers%22):
 
-    <dependency>
-        <groupId>org.testcontainers</groupId>
-        <artifactId>--artifact name--</artifactId>
-        <version>--latest version--</version>
-    </dependency>
+Maven style:
```

Comment:
We should probably mention the Testcontainers BOM here, but can be in another PR, docs have to be reworked after all.